src/main/js/index.js

```diff
@@ -2,3 +2,13 @@ import { defineCustomElements } from "ionicons/dist/loader";
 defineCustomElements(window, {
   resourcesUrl: '/',
 });
+
+import $ from 'jquery';
+import dt from 'datatables.net';
+$(document).ready( function () {
+  if (window.location.pathname === "/probes") {
+    $('#probes-table').DataTable(
+      {"order":[[3, 'asc']]}
+    );
+  }
```

Comment:
This simple complex. Can we not have a `<script>` in the correct template and not have this in the global javascript?

Reply:
Sorry for the late reply, I have been writing the draft proposal with all my strength these two days.

I see what you mean, I'm going to try it further, I also find it weird to put it here